<ui version="4.0" >
 <class>TimeLeaseEditor</class>
 <widget class="QWidget" name="TimeLeaseEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>116</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>TimeLease</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QRadioButton" name="radioFixed" >
       <property name="text" >
        <string>Fix√©e</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioInfinite" >
       <property name="text" >
        <string>Infinie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Valeur fixe</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Jours :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Heures :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Minutes :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QSpinBox" name="spinDays" >
        <property name="maximum" >
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinHours" >
        <property name="maximum" >
         <number>23</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QSpinBox" name="spinMinutes" >
        <property name="maximum" >
         <number>59</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>radioFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>32</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel" >
     <x>62</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
